views/templates/hook/onepagecheckout-p4x.tpl

```diff
@@ -85,10 +86,10 @@
         }, 500);
     }
 </script>
-<span class="pagantisSimulator{$PAGANTIS_CODE|escape:'htmlall':'UTF-8'}></span>
+<span class="pagantisSimulator{$PAGANTIS_CODE|escape:'htmlall':'UTF-8'}"></span>
 <style>
     .pgSimulatorPlaceholder {
         display: inline-block;
     }
-    {$PAGANTIS_SIMULATOR_CSS_CHECKOUT_PAGE_STYLES|escape:'javascript':'UTF-8'}
-</style>
\ No newline at end of file
+    '{$PAGANTIS_SIMULATOR_CSS_CHECKOUT_PAGE_STYLES|escape:'javascript':'UTF-8'}'
```

Comment:
this code cant be between ' because the expected result is something like

.cssRule {
    display:block
}